src/standalone/Vegetation/solar_induced_fluorescence.jl

```diff
@@ -89,6 +153,65 @@ function update_SIF!(p, Y, sif_model::Lee2015SIFModel, canopy)
 end
 
 Base.broadcastable(m::SIFParameters) = tuple(m)
+
+
+"""
+    compute_SIF_at_a_point(
+        APAR::FT,
+        Tc::FT,
+        Vcmax25::FT,
+        Jmax::FT,
+        J::FT,
+        T_freeze::FT,
+        sif_parameters::SIFParameters{FT},
+    ) where {FT}
+    
+Computes the Solar Induced Fluorescence (SIF) at 755 nm in W/m^2 using the Lee et al. 2015 model.
+This function is used for the PModel photosynthesis model, which computes Jmax and J in the cache.
+"""
+function compute_SIF_at_a_point(
```

Comment:
I think it would be clearer to rename this compute_SIF_at_a_point_pmodel. Otherwise we rely on dispatch on the number of arguments to pick the right method.